internal/tools/aws/iam.go

```diff
@@ -21,12 +21,15 @@ type policyDocument struct {
 }
 
 type policyStatementEntry struct {
-	Sid      string
-	Effect   string
-	Action   []string
-	Resource string
+	Sid       string
+	Effect    string
+	Action    []string
+	Resource  string
+	Condition policyStatementCondition `json:"Condition,omitempty"`
 }
 
+type policyStatementCondition map[string]map[string][]string
```

Comment:
`map[string]map[string][]string` hurts me. This type definition would be my suggestion for making it less painful, so, I guess you've already done it. But wow what a type signature. :grimacing:

Reply:
Yeah, it's a mess and I totally agree. Strangely, the policy doc struct I have here is not something that AWS seems to provide in the SDK. Their own example does the same thing in defining it and then parsing it to JSON. Anyway, yeah, maybe something to improve in the future.